community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/spi/CSVResources.scala

```diff
@@ -39,7 +39,10 @@ class CSVResources(cleaner: TaskCloser) extends ExternalResource {
 
   def getCsvIterator(url: URL, fieldTerminator: Option[String] = None): Iterator[Array[String]] = {
     val inputStream = openStream(url)
-    val reader = Readables.wrap(new InputStreamReader(inputStream, "UTF-8"))
+    val reader = Readables.wrap(new InputStreamReader(inputStream, "UTF-8")
```

Comment:
God also states `And don't be afraid of dedicating one line to a curly bracket, for I will be with you and will protect you`